media-plugins/distrho-ports/metadata.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
```

Comment:
Hmm, I'd very much prefer this to be a person. First of all because then we can easily see who the maintainer is. Second of all because I'd like to add a feature that assigns issues relating to packages to the maintainer here on github and we need the info for that.
Also, if we'd add a project here it should be "gentoo-audio", but because of the above IMHO we shouldn't.

Regarding input from others, if there are people who want to contribute please point them to this repo so they can either create an issue here with a description of what they want or, even better, create a PR to implement their fixes or join the team if they want to maintain a package.